packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -242,18 +242,17 @@ class BottomSheet extends Component {
 
 	onSetMaxHeight() {
 		const { height, width } = Dimensions.get( 'window' );
-		const { safeAreaBottomInset } = this.state;
+		//const { safeAreaBottomInset } = this.state;
 		const statusBarHeight =
 			Platform.OS === 'android' ? StatusBar.currentHeight : 0;
 
 		// `maxHeight` when modal is opened along with a keyboard
 		const maxHeightWithOpenKeyboard =
 			0.95 *
-			( Dimensions.get( 'window' ).height -
-				this.keyboardHeight -
+			( height -
+				//this.keyboardHeight -
```

Comment:
This can likely go back in since we just need to tweak the max height calculation below